circle/src/deep_quotient.rs

```diff
@@ -77,18 +80,18 @@ pub(crate) fn deep_quotient_reduce_matrix<F: ComplexExtendable, EF: ExtensionFie
 /// |lde_domain| > |orig_domain|
 #[instrument(skip_all, fields(bits = log2_strict_usize(lde.len())))]
 pub fn extract_lambda<F: ComplexExtendable, EF: ExtensionField<F>>(
-    orig_domain: CircleDomain<F>,
-    lde_domain: CircleDomain<F>,
     lde: &mut [EF],
+    log_blowup: usize,
 ) -> EF {
-    let num_cosets = 1 << (lde_domain.log_n - orig_domain.log_n);
+    let log_lde_size = log2_strict_usize(lde.len());
+    // let num_cosets = 1 << (log_lde_size - orig_domain.log_n);
 
     // v_n is constant on cosets of the same size as orig_domain, so we only have
     // as many unique values as we have cosets.
-    let v_d_init = lde_domain
+    let v_d_init = CircleDomain::<F>::standard(log_lde_size)
```

Comment:
Maybe better clarify the comment in line 89: `v_n ` is constant over twin-cosets of the same size as the original domain, so we only have as many unique values as we have twin-cosets.